src/Maestro/SubscriptionActorService/DarcRemoteFactory.cs

```diff
@@ -83,7 +83,7 @@ public async Task<IRemote> GetRemoteAsync(string repoUrl, ILogger logger)
                     case "github.com":
                         if (installationId == default)
                         {
-                            throw new SubscriptionException($"No installation is avaliable for repository '{normalizedUrl}'");
+                            throw new DependencyException($"No installation is avaliable for repository '{normalizedUrl}'");
```

Comment:
I think we should add a new exception type for this, as it probably could be fatal in some other cases. There may be places we catch DependencyException today that we would not want to catch in this case.

Reply:
I checked and that's not the case, but it makes sense. I'll make it a new exception type.

Reply:
Added a new type for issues with the Github App installation for future-proofing.